README.md

```diff
@@ -34,21 +34,21 @@ These three libraries provide a powerful combination of HPC message passing tool
 
 ### dask ###
 
-    git clone git@github.com:rapidsai/dask.git
+    git clone https://github.com/dask/dask.git
     cd dask
     pip install -e .
     cd ..
 
 ### dask distributed ###
 
-    git clone git@github.com:dask/distributed.git
+    git clone https://github.com:dask/distributed.git
     cd distributed
     pip install -e .
     cd ..
 
 ### dask-cuda ###
 
-    conda install -c rapidsai dask-cuda
+    conda install -c rapidsai-nightly dask-cuda
```

Comment:
I think we would have to add other channels here too as this still seems to pulling a version from `19-09-24`
i.e it pulls below version:
```rapidsai-nightly/label/cuda10.0/linux-64::dask-cuda-0.10.0a190924-py37_15```

While below pulls the latest version:

```bash
conda install -c rapidsai-nightly/label/cuda10.0 -c rapidsai -c nvidia -c numba -c conda-forge dask-cuda
```

Reply:
I don't think we want to use the CUDA labeled channels. Those are getting phased out. Also it locks in a specific CUDA Version, which the user may not want.

My guess is if this was needed there is another issue occurring here like packages are missing or something similar. Could you please raise a separate issue about this? Possibly on dask-cuda?

Reply:
Yup, 
I forgot to remove the `cuda` version. 
Below works too. 

```
 conda install -c rapidsai-nightly -c rapidsai -c nvidia -c numba -c conda-forge dask-cuda --dry
```
Sure, I will raise the issue on  `dask-cuda` .

Reply:
Raised: https://github.com/rapidsai/dask-cuda/issues/159

Reply:
Thanks @VibhuJawa! 😄